LanguageFeatures/Extension-types/static_analysis_member_invocation_A06_t01.dart

```diff
@@ -2,11 +2,12 @@
 // for details. All rights reserved. Use of this source code is governed by a
 // BSD-style license that can be found in the LICENSE file.
 
-/// @assertion We say that an extension type declaration DV has a non-extension
-/// type member named n in the case where DV does not declare a member named n,
-/// but DV has a direct extension type superinterface V that has a non-extension
-/// type member named n, or DV has a direct non-extension type superinterface T
-/// whose interface contains a member signature named n.
+/// @assertion We say that an extension type declaration DV has an extension
+/// type member named n in the cases where:
+/// - DV declares a member named n.
+/// - DV has no such declaration, but DV has a direct extension type
+///   superinterface V that has an extension type member named n due to a member
+///   declaration DM2, and DV does not declare a member that precludes DM2.
 ///
 /// @description Checks that if an extension type `ET` has a superinterface with
 /// a member `m` then this member is also presents in `ET`
```

Comment:
I think we have a number of occurrences of this:
```suggestion
/// a member `m` then this member is also present in `ET`
```